src/status_im2/contexts/contacts/events.cljs

```diff
@@ -0,0 +1,22 @@
+(ns status-im2.contexts.contacts.events
+  (:require
+   [utils.re-frame :as rf]
+   [status-im.contact.db :as contact.db]
+   [status-im.data-store.contacts :as contacts-store]))
+
+(rf/defn load-contacts
+  {:events [:contacts/contacts-loaded]}
+  [{:keys [db] :as cofx} all-contacts]
+  (let [contacts-list (map #(vector (:public-key %)
```

Comment:
@alwx, FYI:

We had quite a few discussions in the team about what should be tested in the re-frame layer and at least for now, we've settled on testing layer-3 subscriptions and, ideally, event handlers.

From the official docs:

>- Event Handlers - most of your testing focus will be here because this is where most of the logic lives
>- Subscription Handlers - often not a lot to test here. Only [Layer 3](https://github.com/day8/re-frame/blob/master/docs/subscriptions.md) subscriptions need testing.
>
>-- https://github.com/day8/re-frame/blob/master/docs/Testing.md#what-to-test

I know this code already existed and you just moved it. If this event handler was a new implementation, I'd ask you to please test it, or at least extract the logic that builds the `contacts-list` and unit test it.

The reasoning is simple: if we can make sure the application state is properly tested, a lot of bugs will be avoided and it'll be easier for everyone to evolve the product.

Reply:
We also have `data_store/` directory, where this kind of logic should be extracted in case you want to proceed with the refactor.

Reply:
I want to proceed with refactor but probably not as a part of this task :)